packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -156,6 +164,20 @@ - (UIView *)createRootViewWithBridge:(RCTBridge *)bridge
   return rootView;
 }
 
+- (void)_logWarnIfOverridden
```

Comment:
nit: How about more descriptive naming? `_logWarnIfCreateRootViewOverridden` or something like this